src/pymovements/dataset/dataset_files.py

```diff
@@ -336,7 +336,7 @@ def load_gaze_file(
             column_dtypes=definition.filename_format_dtypes,
         )
     elif filepath.suffix == '.asc':
-        gaze_df = from_asc(
+        gaze_df, _ = from_asc(
```

Comment:
here we could event think about reusing this at some point e.g. to extract label data

Reply:
By "label data", do you mean things like responses to comprehenion questions?

Reply:
yes

Reply:
but future music.